cpp_code/include/CrossCat/RandomNumberGenerator.h

```diff
@@ -20,26 +20,24 @@
 #ifndef GUARD_randomnumbergenerator_h
 #define GUARD_randomnumbergenerator_h
 
-#include <boost/random/mersenne_twister.hpp>
-#include <boost/random/uniform_01.hpp>
 #include <ctime>
-//
+#include <stdint.h>
+
 #include "constants.h"
+#include "weakprng.h"
 
-/////////////////////////
-// from runModel_v2.cpp
 class RandomNumberGenerator {
 public:
-    RandomNumberGenerator() : _engine(0), _dist(_engine) {};
-    RandomNumberGenerator(int SEED) : _engine(SEED), _dist(_engine) {};
+    RandomNumberGenerator(int seed = 0) { set_seed(seed); }
     double next();
-    int nexti(int max = MAX_INT);
+    int nexti(int bound = MAX_INT);
+    double stdnormal();
+    double stdgamma(double alpha);
```

Comment:
And for posterity's sake: Done in 267ccd8cf62ddaeb0a8841577c8ae0fec675cd96, 473c7902e372b628be32e1966b7efacdf9d5c24f, 81da0c6091779903195c1b82039eb6e60f907c9f, 8c2a71e9037d760389a65b9816e9085f471c72d4, 531934de397afe1dab55f890f62674b0ded847ff, 28ec06ff402292012429362f17c652d1309694e2.